wiki/osu!stream/en.md

```diff
@@ -91,8 +90,6 @@ In 2023, twelve years after its initial release, osu!stream was finally [ported
 
 ![](img/Os-HC.jpg "Hold circle 1")
 
----
```

Comment:
Do you want to keep these for visual separation?

Reply:
feels like this should be okay either way